test/cypress/tests/modules/BlockEvents/Slash.cy.ts

```diff
@@ -92,6 +92,69 @@ describe('Slash keydown', function () {
         .should('eq', 'Hello/');
     });
   });
+
+  describe('pressed outside editor', function () {
+    it('should not modify any text outside editor when text block is selected', () => {
+      // Create editor with an empty block
+      cy.createEditor({
+        data: {
+          blocks: [
+            {
+              type: 'paragraph',
+              data: {
+                text: '',
+              },
+            },
+          ],
+        },
+      });
+
+      // First click the plus button to open the toolbox
+      cy.get('[data-cy=editorjs]')
+        .find('.ce-paragraph')
+        .click();
+
+      cy.get('[data-cy="toolbox"] .ce-popover__container')
+        .should('not.be.visible');
+
+      // Get the heading text content before the slash key press
+      cy.get('h1')
+        .contains('Editor.js test page')
+        .invoke('text')
+        .then((originalText) => {
+          // Simulate selecting the heading text
+          cy.get('h1')
+            .contains('Editor.js test page')
+            .trigger('mousedown')
+            .trigger('mouseup');
```

Comment:
This code does not work for me. 

I've commented out changes in `blockEvents.ts` but the text in H1 still not being removed. It is expected to see reproduction of the original problem in this test case.